profiles/shivam-jha2712.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "![Screenshot (1298)](https://user-images.githubusercontent.com/71783722/193400453-e7624e82-9d78-45b1-80e1-d20876e46ea5.png)",
```

Comment:
```suggestion
  "image": "https://user-images.githubusercontent.com/71783722/193400453-e7624e82-9d78-45b1-80e1-d20876e46ea5.png",
```

Reply:
Could you please let me know if I should go for a pull request again by making the changes or how I should edit the current pull request if YES? How to do so

Reply:
I think you did it right 👍